### PR TITLE
Create graph from scratch in content nightly

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-9667)

## Description
Set `DEMISTO_SDK_GRAPH_FORCE_CREATE` to `true` in nightly to create the graph from scratch in order to avoid failures of incompatible content.

[Nightly](https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/pipelines/823861)

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by ilaner — APPROVED** [review]
nice :)

**Comment by ilaner** [comment]
please trigger content nightly with this branch and paste the link to the description

**Comment by MosheEichler** [comment]
> please trigger content nightly with this branch and paste the link to the description

[Nightly](https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/pipelines/812382)

File: .gitlab/ci/.gitlab-ci.global.yml [hunk 1]
```diff
@@ -132,6 +132,11 @@
   - chmod +x ./Tests/scripts/*
   - chmod +x ./Tests/Marketplace/*
   - section_end "Granting execute permissions on files"
+  # DEMISTO_SDK_GRAPH_FORCE_CREATE set to true to create graph from scratch on last upload commit
+  - |
+    if [[ -n $NIGHTLY ]]; then
```
Comment (ilaner) [inline .gitlab/ci/.gitlab-ci.global.yml:137]:
```suggestion
    if [[ $NIGHTLY ]]; then
```

**Comment by MosheEichler** [comment]
> [.gitlab/ci/.gitlab-ci.global.yml](/demisto/content/pull/32766/files/96e9e419ec8f209944841af0430bf7a49185f068#diff-224bd41c2749ef01f62e1efabd3509b1fbd9e6f0f8b6edd32dd5d6107a09c14a)

[re-run nightly](https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/pipelines/814905)

**Pull request merged**